src/app/portlets/dot-edit-page/layout/dot-edit-layout-designer/dot-edit-layout-designer.component.spec.ts

```diff
@@ -334,6 +334,24 @@ describe('DotEditLayoutDesignerComponent', () => {
                 expect(dotDialogService.alert).not.toHaveBeenCalled();
                 expect(component.setEditLayoutMode).toHaveBeenCalled();
             });
+
+            it('when editTemplate Input  id different than true', () => {
```

Comment:
extra spaces... is not "Input id" is an component attribute and different that true? that's why I think it should be false by default so we make sure is 100% a boolean.